docs/console-api/experiments.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Experiments
+sidebar_label: Experiments
+slug: /console-api/experiments
+description: Create feature experiments via HTTP requests.
+hide_table_of_contents: true
+custom_edit_url: null
+---
+import Rapidoc from "../../src/components/Rapidoc"
+
+<Rapidoc id="experiments" entity='experiments'/>
```

Comment:
wow this is so cool. Now you can just specify the schema and it generates a doc page from that?  Gamechanger

Reply:
<3